frontend/integration-tests/tests/environment.scenario.ts

```diff
@@ -80,9 +81,16 @@ describe('Interacting with the environment variable editor', () => {
     }
   };
 
-  const validateValueFrom = async (valueFrom: string, prefix: string) => {
-    expect(environmentView.resources.last().getText()).toEqual(valueFrom);
-    expect(environmentView.prefix.getAttribute('value')).toEqual(prefix);
+  const validateValueFrom = async (valueFrom: string, prefix: string, isPresent: boolean) => {
+    const resourceText = await environmentView.resources.last().getText();
+    const prefeixText = await environmentView.prefix.getAttribute('value');
+    if (isPresent) {
+      expect(resourceText).toEqual(valueFrom);
+      expect(prefeixText).toEqual(prefix);
+    } else {
+      expect(resourceText).toEqual('hello-openshift');
```

Comment:
`isPresent:false` should equal `hello-openshift` seems a little strange to me.

Reply:
yeah, it is very strange, that is what the container name is in the first resource text box.  The components  have the same name.  I should probably tweak that.

Reply:
So, I can rename things or add a data id, but I think for now I would prefer to leave it so I don't have to alter too much code, especially since this test is on the way out.

Reply:
Works for me till @dtaylor113 and our Cypress stuff is ready to roll.

/approve
/lgtm